consensus/istanbul/backend/handler.go

```diff
@@ -381,7 +381,7 @@ func (sb *Backend) RegisterPeer(peer consensus.Peer, isProxiedPeer bool) error {
 		}
 	}
 
-	if err := sb.sendVersionCertificateTable(peer); err != nil {
+	if err := sb.announceManager.SendVersionCertificateTable(peer); err != nil {
```

Comment:
thinking out loud on alternative "semantics". We could do:


1) announce doesn't  know about this, we do it: 
table = announceManager.getTable(); peer.send(table)

2) announce knows about this
announceManager.handleRegisterPeer(peer)

Reply:
Mhm. I think I like the second one better. In the future PRs (where everyting is moved into the AnnounceManager) I'll explore actual refactors to the interface instead of just moving pieces around